qcodes/instrument_drivers/Keysight/keysightb1500/KeysightB1500_module.py

```diff
@@ -212,6 +217,18 @@ def get_name_label_unit_of_impedance_model(
 #   it might make more sense to generate one for each **channel**
 
 
+def get_measurement_summary(status_array: np.ndarray) -> str:
+    unique_error_statuses = np.unique(status_array[status_array != "N"])
+    if len(unique_error_statuses) > 0:
+        summary = " ".join(
+            constants.ComplianceStatus[err] for err in
+            unique_error_statuses
+        )
+    else:
+        summary = constants.ComplianceStatus["N"]
```

Comment:
is ComplianceStatus a coerect name? per my memory those statuses cover more than just compliance but i might be misremembering. i'd suggest MeasurementStatus, a bit broader.

Reply:
Ok. I changed to MeasurementStatus